### PR TITLE
Add call to action to talk with participants

### PR DESCRIPTION
**Pull request by ernesto-jimenez**
Included a call to action in the home page to talk with other
participants via the twitter hashtag or the Slackline channel.

### CODE REVIEW

**Comment by ernesto-jimenez** [comment]
@agonzalezro here's the PR as discussed :)

Here's how it looks like.
screen shot 2015-08-19 at 12 05 47

I would add the CTA to the top of the schedule page too. It's probably going to be the most visited page this week.

Do you want me to do that?

File: index.html [hunk 1]
```diff
@@ -19,8 +19,11 @@
 					<div class="subscribe">
 						<div style="width:100%; text-align:left;" ><iframe  src="//eventbrite.co.uk/tickets-external?eid=16233380478&ref=etckt" frameborder="0" style="min-height:375px;" width="100%" vspace="0" hspace="0" marginheight="5" marginwidth="5" scrolling="no" allowtransparency="true"></iframe></div>
 					</div>
+					<h3>
+						Meet other participants. Follow the <a href="https://twitter.com/search?q=%23golanguk15" target="_blank">#golanguk15 tweets</a> or get on the <a href="https://slackline.io/shared_channels/golang-uk-conf" target="_blank">Golang UK channel in Slackline</a>
```
Comment (johannac) [inline index.html]: The hashtag is #gouk15

**Comment by agonzalezro** [comment]
Change the hashtag and it will be ok.

I wonder if there is a better way to show this, like perhaps moving the phrase before the ticket selling. Any opinion on this @johannac, @dahernan || @samdb?

We should publish this on twitter soon. Perhaps today so the people perhaps can meet before the conference (if they are not Londoners).

**Comment by ernesto-jimenez** [comment]
@agonzalezro @johannac, updated the hashtag

**Comment by ernesto-jimenez** [comment]
If you want me to move add the text to the schedule page too and/or text before the ticket selling just let me know :)

**Pull request merged**